features/step-definitions/boards.steps.js

```diff
@@ -0,0 +1,106 @@
+const { Given, When, Then } = require("@wdio/cucumber-framework");
```

Comment:
Steps should be composed not using PO approach, but the other - steps for actions, steps for validations ...
so you should reorganize all your steps files.
(you've probably noticed that you have 2 the same steps - the user is logged in and on the main page AND the user is logged into Trello. it's caused by wrong composition)